src/commands/token/refreshTokenCommand.ts

```diff
@@ -0,0 +1,49 @@
+import { RefreshTokenInterface } from "../../interfaces/token";
+
+export default class RefreshTokenCommand {
```

Comment:
same here, let's just create an interface and pass an object around, like authorizationCommand